ddtrace/tracer/span.go

```diff
@@ -319,6 +323,12 @@ func (s *span) Finish(opts ...ddtrace.FinishOption) {
 		s.taskEnd()
 	}
 	s.finish(t)
+
+	// Restore the labels of the parent span so any CPU samples after this point
```

Comment:
Can we do the same with the comment here? Put it inside the `if` clause as the style guide mentions.